.claude/skills/changelog/SKILL.md

```
@@ -0,0 +1,92 @@
+
+#### Changed
+- <entry>
+
+#### Fixed
+- <entry>
+
+### Skipped
+- <merge commits, version bumps, etc.>
+```
+
+## Rules
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)